newdle/core/util.py

```diff
@@ -1,11 +1,12 @@
 from datetime import datetime
 
-import pytz
 from sqlalchemy import func, types
 from sqlalchemy.sql import operators
 from sqlalchemy.sql.sqltypes import Interval
 from sqlalchemy.util import memoized_property
 
+import pytz
```

Comment:
What's the reason of this change? I think it was OK.

Reply:
That looks like someone ran isort without the package being properly installed